README.md

```diff
@@ -32,10 +32,11 @@
   <a href="https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt">
     PGP
   </a>
-</div>
+</h3>
 
 <br>
```

Comment:
please remove this and the next line
it's causing unnecessary spacing.

then it's good to go

Reply:
Thanks. Please review again